### PR TITLE
Remove COMPNAME

### PR DESCRIPTION
**Pull request by tristan957**
This is an addendum to Greg's earlier commit.

Signed-off-by: Tristan Partin <tpartin@micron.com>

## Verified checks?
Have the checks completed?
- [x] meson test -C build --setup=ci
- [x] All commits are signed off

### CODE REVIEW

File: cli/meson.build [hunk 1]
```diff
@@ -15,9 +15,8 @@ hse_cli_sources = files(
     'storage_profile.c',
 )
 
-hse_cli_c_args = [
-    '-DCOMPANE="CLI"',
```
Comment (tristan957) [inline cli/meson.build:19]: This is kind of funny. Was probably broken for the past 7 months.

**Review by beckerg — APPROVED** [review]

**Comment by beckerg** [comment]
Way too much overhead to approve these tiny PRs, I see no reason you could combine this change with any other PR...

**Comment by tristan957** [comment]
Then the git history is pointless. I brought this up in previous meetings as did John.

**Comment by nabeelmmd** [comment]
> Then the git history is pointless. I brought this up in previous meetings as did John. This is the pro/con of Squash and Merge. You get tiny pull requests if you want to keep the git history clean at the expense of requiring someone to hit approve on all those tiny PRs.
> 
> Just a sign that there aren't enough people on this project, which we all know already.

@tristan957 , not sure I follow your point regarding git history. What is the issue if you squashed all the changes from these tiny pull requests in your local branch and created a single PR for it? It would still build and run fine, right? If you would like to keep track of the changes, a descriptive commit message listing all the modifications would suffice, I think. Probably, I'm missing something.

**Review by nabeelmmd — APPROVED** [review]

**Comment by tristan957** [comment]
Maybe I am missing something, but with Squash and Merge, I understand it to be that all the commits in a PR get squashed into 1 single commit, which loses all the detailed commits along the way. This is fine if all your commits are tiny fixup commits, but sucks when your commits are all individual changes that can stand completely on their own.

**Pull request merged**